### ISSUE TITLE
Issue with pytorch version

### ISSUE THREAD

**Issue by ibrahim-601** [zhou13/lcnn#53]
I tried following the instructions on the repository. Pytorch 1.8.1 is installed with the instructions. But with that environment, the right points of lines get a little bit downward. But if I try the same with PyTorch 1.4.0 there is no issue. Can you please explain why is this happening? Or can you update the instructions for the correct PyTorch version? I have attached both outputs. The first one is with PyTorch 1.4.0 and the second one is with PyTorch 1.8.1.

PyTorch 1.4.0 sample:

![table_torch_1 4 0](https://user-images.githubusercontent.com/50712343/134464955-e219d2fd-1304-462e-85de-cc157b44f5c5.jpg)

PyTorch 1.8.1 sample:
![table_torch_1 8 1](https://user-images.githubusercontent.com/50712343/134464965-28241c3c-3341-4926-9efd-3ae7d74981ee.jpg)

**Comment by zhou13** [comment]
Hi @ibrahim-601, Thank you for reporting. This is an interesting finding. Could you reproduce the result using `demo.py` with the pre-trained model? Could you also upload the original image in PNG so I can investigate? If you can help to debug, could you help print/plot             `jmap` and `joff` at https://github.com/zhou13/lcnn/blob/88f281ab5421d51a62f1f84f97fea05afbf0c8d8/lcnn/models/line_vectorizer.py#L162 and see if they are the same in Python 1.8.1 and Python 1.4.0?

**Comment by ibrahim-601** [comment]
Hi @zhou13, thank you very much for replying. The above result is produced with `demo.py` with the pre-trained model. I saved the `jmap` and `joff` values in a JSON file using below code
```
js_data = {'jmap_after_nms': jmap.cpu().numpy().tolist(), 'joff': joff.cpu().numpy().tolist()}
json.dump(js_data, open('jmap_joff.json','w'))
```
at https://github.com/zhou13/lcnn/blob/88f281ab5421d51a62f1f84f97fea05afbf0c8d8/lcnn/models/line_vectorizer.py#L162
The values of `jmap` and `joff` in PyTorch 1.4.0 and PyTorch 1.8.1 are the same.
I have uploaded the original image and some test results along with the JSON files [here](https://drive.google.com/drive/folders/1wvI7wpbEViVl75-TGBllm0OD_1cUSZLn?usp=sharing).
